.gitignore

```diff
@@ -34,12 +34,4 @@
 /upgrade
 /upgrade_errors.txt
 /translations
-/media/images/*
-!/media/images/flags
-!/media/images/apple-touch-icon.png
-!/media/images/avatar.png
-!/media/images/favicon.ico
-!/media/images/mautic_logo*
-!/media/images/.htaccess
-/media/dashboards/*
-!/media/dashboards/*.json
\ No newline at end of file
+/media/*
```

Comment:
Sorry, don't have a reason. Was a test that I made, nothing to do with the project, and the .gitignore was commited together and I didn't see.